.github/workflows/release-request.yml

```diff
@@ -0,0 +1,35 @@
+name: Open release request
+
+on:
+  workflow_dispatch:
+    inputs:
+      gems:
+        description: Gems to release (leave blank to release all gems)
+        required: false
+        default: ""
```

Comment:
Does this have to be manually typed in? If we need to release, say, the Jaeger exporter and the MySQL adapter, what is the syntax required here?

Reply:
```
opentelemetry-exporter-jaeger, opentelemetry-instrumentation-mysql2
```

The syntax is actually:

```
VERSEP = :
GEMSEP = /[,\s]+/
GEMNAME (VERSEP VERSION)? (GEMSEP GEMNAME (VERSEP VERSION)?)*
```

So you can specify versions for the gems if you want, or you can let the release scripts figure out the version based on conventional commits and semver. I guess this probably needs to be covered in the CONTRIBUTING doc. I'll add it.